vcr.gemspec

```diff
@@ -22,4 +22,6 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = ">= 2.7"
 
   spec.add_dependency "base64"
+
+  spec.metadata["changelog_uri"] = "https://github.com/vcr/vcr/blob/master/CHANGELOG.md"
```

Comment:
@andrehjr Personally I am generally happy with the most recent file since it includes all the previous revisions in it, but I get that the per-release link is also popular. I'm very happy to go with the majority view.